Configuration/Geometry/python/GeometryExtended2023Reco_cff.py

```diff
@@ -15,13 +15,12 @@
 #Muon
 from Geometry.MuonNumbering.muonNumberingInitialization_cfi import *
 from RecoMuon.DetLayers.muonDetLayerGeometry_cfi import *
-from Geometry.GEMGeometry.gemGeometry_cfi import *
```

Comment:
@dildick - thanks! I'll add them. However, the 2023 scenario does not have ME0 in. Should it be added as well?